explorer/pages/explore/files/[uuid].tsx

```diff
@@ -1,9 +1,9 @@
 import React from "react";
```

Comment:
Go ahead and delete the file and sample detail pages for now as we won't have detail pages for these at the moment.